src/components/App.js

```diff
@@ -36,9 +44,13 @@ class App extends React.Component {
           <div className="slate">
             <h2> Score </h2>
             <ScoreBoard></ScoreBoard>
+            <h2> Room Info </h2>
+            <p> Room ID : {room}</p>
             <h2> Letters </h2>
             <Slate></Slate>
+            <button onClick = { this.onSubmit }> See Game Rules </button>
```

Comment:
This button can be separated from the board component and made generic based on the state, ie, it can show "view game rules" when isBoardVisible is true and can show "return to board" when false.
A separate button is not needed in Rules

Reply:
See when rules component loads,If there isn't a button then how do I change the state of isRulesVisible from App component.

Reply:
That's why I asked you to separate the button from both the individual components and have one common button for the state toggling.

Reply:
And where that common button lives in app.js?

Reply:
Inside <App> and outside <Slate> and <Board> ?